tests/e2e/utils.go

```diff
@@ -90,3 +97,23 @@ func getActiveApplication() string {
 	odoActiveApp := strings.Split(reActiveApp.FindString(stdOut), "*")[1]
 	return strings.TrimSpace(odoActiveApp)
 }
+
+func retryingForOutputMatchStringOfHTTPResponse(url, match string, retry, sleep int) bool {
```

Comment:
Please add a comment describing this function and its arguments.

Once you have a description of what this function is doing in a comment instead of its name :-) 
you can make the name of this function a little bit shorter ;-) Long names like this are really hard to read